src/context-providers/yup/helper.ts

```diff
@@ -237,8 +237,8 @@ export namespace YupHelper {
 			const customRuleKey = Object.keys(rule).filter((k) =>
 				customYupConditions.includes(k as TYupCondition)
 			)?.[0] as TYupCondition;
-			if (customRuleKey) {
-				yupSchema = (yupSchema as unknown)[customRuleKey]?.(rule[customRuleKey], rule.errorMessage);
+			if (customRuleKey && (yupSchema as unknown)[customRuleKey]) {
+				yupSchema = (yupSchema as unknown)[customRuleKey](rule[customRuleKey], rule.errorMessage);
 			}
```

Comment:
can we follow the preceding codes and do `console.warn()` to inform the consumer that the rule is not applied?

```
console.warn(`error applying "${customRuleKey}" condition to ${yupSchema.type} schema`);
```